src/haddock/libs/libplots.py

```diff
@@ -1120,15 +1206,15 @@ def report_generator(boxes, scatters, tables, step):
         report.write(html_report)
 
 
-
 def heatmap_plotly(
         matrix: NDFloat,
         labels: Optional[dict] = None,
         xlabels: Optional[list] = None,
         ylabels: Optional[list] = None,
         color_scale: str = 'Greys_r',  # Greys_r, gray
         title: Optional[str] = None,
-        output_fname: Path = Path('contacts.html'),
+        output_fname: Path = HEATMAP_DEFAULT_PATH,
+        offline: bool = False,
```

Comment:
Why are even the labels optionals but this is not? here and in many other functions